docs-markdown/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Docs Markdown Extension",
   "icon": "images/docs-logo-ms.png",
   "aiKey": "0a0e5961-85c2-451a-bce8-6a54e37c93be",
-  "version": "0.2.37",
+  "version": "0.2.38",
```

Comment:
@jamarw - Believe it or not, we need to bump this up :). We just merged in a hotfix.

Reply:
@adunndevster , thanks for the heads up!